src/themes/uswitch-rebrand/theme.json

```diff
@@ -778,16 +933,25 @@
             "borderBottomWidth": 4,
             "borderBottomStyle": "solid",
             "borderBottomColor": "primary",
-            "::before":{
+            "::before": {
               "content": "''",
               "position": "absolute",
               "width": 24,
               "height": 24,
-              "left": ["calc(50% - 24px/2)", -12],
-              "top": [-10, "calc(50% - 18px)"],
-              "background": "white",
+              "left": [
+                "calc(50% - 24px/2)",
+                -12
+              ],
+              "top": [
+                -10,
+                "calc(50% - 18px)"
+              ],
+              "background": "light-1",
```

Comment:
`background` isn't swapped with colours from the theme, only `backgroundColor` is. Previously this value was `white` which is a CSS value so it worked even though it wasn't being updated with the value from the theme.